sdk-actors/src/main/java/io/dapr/actors/runtime/ReminderOptions.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright 2022 The Dapr Authors
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package io.dapr.actors.runtime;
+
+import java.time.Duration;
+
+public class ReminderOptions {
+    
+  //required parameters
+  private String name;
+
+  //optional parameters
+  private Duration dueTime;
+  private Duration period;
+    
+
+  public String getName() {
+    return name;
+  }
+
+  public Duration getDueTime() {
+    return dueTime;
+  }
+
+  public Duration getPeriod() {
+    return period;
+  }
+    
+  private ReminderOptions(ReminderOptionsBuilder builder) {
+    this.name = builder.name;
+    this.dueTime = builder.dueTime;
+    this.period = builder.period;
+  }
+    
+  //Builder Class
+  public static class ReminderOptionsBuilder<T> {
```

Comment:
I think we specifically moved away from boiler pattern in java sdk for model classes https://github.com/dapr/java-sdk/issues/587. 
Can this be changed to reflect that?